### DISCUSSION TITLE
请问如何用AE导出pag sdk version为2或者3的文件？

### DISCUSSION THREAD

**Discussion by casssssius** [Tencent/libpag#2187] [General] [answered]
【背景】因为我们的SDK现在只支持4以下的PAG文件
【问题】想请教下如何产出sdk version是2或者3的pag文件哇？

**Comment by kevingpqi123** [comment, marked as answer]
让你们的设计师关注下 pag 的 AE 特性支持列表，https://pag.art/zh-CN/feature.html#ae
上面描述了每种 AE 特性的支持版本，避免使用 4.0 及以上版本的特性即可，交付之前通过桌面端预览工具查看最低支持版本确认

导出插件层面没有提供特定版本的过滤功能，对于 pag sdk 而言，始终会保持对之前版本的兼容，我们推荐各业务尽可能的升级到最新版本。

**Reply by fiona119119** [reply]
如何查看最低支持版本呢，有没有什么插件可以检验出pag是否用了4.0及以上版本的特性呢

**Reply by domchen** [reply]
> 如何查看最低支持版本呢，有没有什么插件可以检验出pag是否用了4.0及以上版本的特性呢

在SDK里读取PAG文件的PAGFile.tagLevel属性就行。跟你当前SDK的PAGFile::MaxSupportedTagLevel()对比，比它高就是使用了超过了你当前SDK版本支持的特性。
